src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -228,11 +230,28 @@ typedef struct {
     UINT32 selectedCandidatePairChanges;      //!< The number of times that the selected candidate pair of this transport has changed
 } RtcTransportStats, *PRtcTransportStats;
 
+// https://www.w3.org/TR/webrtc-stats/#dom-rtcrtpstreamstats
```

Comment:
Check the formatting once.

Reply:
what do you mean?

Reply:
I mean in this diff, it looks like the comments are not aligned. But, it could just be a diff thing. So, just putting it out here to make clang-format check is done.

Reply:
reformatting doesnt change anything on my side

Reply:
Then feel free to ignore my comment :)